dist/gmusic-theme.js

```diff
@@ -1665,15 +1674,40 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
         }
 
         /**
-         * Enables the custom theme
-         */
+        * Enabled the dark theme, this allows for backwards compatibility
+        */
 
       }, {
         key: "enable",
         value: function enable() {
-          document.body.classList.add(CONSTANTS.CLASS_NAMESPACE);
-          this.enabled = true;
-          this._drawLogo();
+          this.enableAll();
+        }
+
+        /**
+         * Enables the custom theme in dark mode (All colors)
+         */
+
+      }, {
+        key: "enableAll",
+        value: function enableAll() {
+          this.disable();
+          document.querySelector('html').classList.add(CONSTANTS.CLASS_NAMESPACE);
```

Comment:
How come the switch from `body` to `html`, out of interest?

Just wondering if this has the potential to be a breaking change, in the unlikely event that anyone's extending it and targetting `.gmusic-theme.material` on the body tag